telethon/_client/messages.py

```diff
@@ -738,3 +738,41 @@ async def _pin(self, entity, message, *, unpin, notify=False, pm_oneside=False):
 
     # Pinning a message that doesn't exist would RPC-error earlier
     return self._get_response_message(request, result, entity)
+
+async def send_reaction(
+    self: 'TelegramClient',
+    entity: 'hints.EntityLike',
+    message: 'hints.MessageIDLike',
+    reaction: str = None
```

Comment:
I think the type hint of `reaction` should be made `Optional[str]` if you can set it to `None`. 

Otherwise the default value could be an empty string

Reply:
😅 I wrote it None, so that, if it is None, we react using the quick reaction set by us😬